src/commands/init.ts

```diff
@@ -71,6 +72,15 @@ const choices = [
     checked: false,
   },
 ];
+
+if (isEnabled("internaltesting")) {
+  choices.push({
+    value: "frameworkstack",
```

Comment:
let's use the name `frameworks` more consistently. I seem `frameworkstack` and `frameworkstackapi` and would prefer seeing `frameworks`

Reply:
done